haiku-games/matchingcomulns/matching_columns-0.1.0.recipe

```diff
@@ -0,0 +1,42 @@
+SUMMARY="A matching game"
+DESCRIPTION="MatchColumns, a fun matching game."
```

Comment:
This one should serve as a better `DESCRIPTION`

From [spiceweasel/matching-columns](https://github.com/spiceweasel/matching-columns):

> A game for the Haiku OS where users score points by matching the columns before breaking them.

Reply:
Yep, that's a nice description, but I'd change a small thing: `for the Haiku OS` -> `for Haiku`

@alaviss, I think the Haiku community prefers to call the system just "Haiku",  without the "OS".
(I also think it sounds better.)
We even say that "Haiku was inspired by BeOS".
The fact that the website address is haiku-os.org is an "accident": when the name was chosen, haiku.org was already taken and I guess their owners did not seem to be willing to give it for free (or for a friendly price :)